deploy/kiali/kiali_cr.yaml

```diff
@@ -228,6 +228,15 @@ spec:
 #      api_version: "autoscaling/v2beta2"
 #      spec: {}
 #
+# hostAliases setting that will be configured in kiali container.
+# A typical way to configure hostAliases for Kiali is:
+#
+#    host_aliases:
+#      - ip: 192.168.1.100
+#        hostnames:
+#          - "foo.local"
+#          - "bar.local"
+#
```

Comment:
It is ok to have an example yaml here, but this example kiali_cr.yaml file is mainly used by our users to know what the defaults are if you do not specify this value (all values you see in here are the actual defaults if not otherwise specified). See [here](https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L16-L19). So you need to indicate that the default value for `host_aliases` is `[]`.

Since this host_aliases is simply just copied into the Deployment as the Kubernetes hostAliases yaml, I would point to the Kubernetes docs, too.

For consistency, do not add the extra indentation for array items (the kiali syntax standard is to not add the extra indentation in yaml for array list items).

Finally, the Kiali setting is `host_aliases` (i.e. uses snake_case) so if you mention `hostAliases` (the camelCase version) make sure it is clear to the reader that is the k8s yaml setting.

So, based on the above, my suggestion is:

```suggestion
# This is content for the Kubernetes hostAliases setting for the Kiali server.
# This allows you to modify the Kiali server pod /etc/hosts file.
# A typical way to configure this setting is:
#
#    host_aliases:
#    - ip: 192.168.1.100
#      hostnames:
#      - "foo.local"
#      - "bar.local"
#
# For details on the content of this setting, see:
# https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/#adding-additional-entries-with-hostaliases
#    ---
#    host_aliases: []
```

Reply:
Thanks for pointing out, great suggestion!